.travis.yml

```diff
@@ -1,3 +1,14 @@
 language: node_js
 node_js:
   - '4'
+
+after_success:
+  - npm run build
+
+deploy:
```

Comment:
temp configuration.
A github token would be necessary to in order to push an updated readme back.